src/Mod/Sketcher/Gui/SketcherSettings.cpp

```diff
@@ -482,12 +485,14 @@ void SketcherSettingsDisplay::onBtnTVApplyClicked(bool)
                                 "        sketch.ViewObject.ShowSupport = %s\n"
                                 "        sketch.ViewObject.RestoreCamera = %s\n"
                                 "        sketch.ViewObject.ForceOrtho = %s\n"
+                                "        sketch.ViewObject.AlignView = %s\n"
```

Comment:
I wonder if that code is actually still needed? `ViewProviderSketch` does implement parameter observer that should do just that... Nonetheless, this is refactor opportunity and not something for that PR :)